doc/source/whatsnew/v0.21.0.txt

```diff
@@ -40,6 +40,8 @@ Other Enhancements
 - :func:`DataFrame.clip()` and :func:`Series.clip()` have gained an ``inplace`` argument. (:issue:`15388`)
 - :func:`crosstab` has gained a ``margins_name`` parameter to define the name of the row / column that will contain the totals when ``margins=True``. (:issue:`15972`)
 - :func:`Dataframe.select_dtypes` now accepts scalar values for include/exclude as well as list-like. (:issue:`16855`)
+- It is now possible to read data (i.e. CSV, JSON, HTML) from a HTTP/HTTPS URL that requires Basic Authentication (:issue:`16716`)
+- It is now possible to skip check of validity of SSL certificate (eg: Testing using self-signed SSL certificates) (:issue:`16716`)
```

Comment:
We should condense into one line:
~~~
- It is possible to read data (i.e. CSV, JSON, HTML) from
a URL that is password-protected (:issue:`16716`)
~~~

Add a section right under "Other Enhancements" with my shortened description followed by a longer description about what you can do now with authentication.  Try to follow what was done with "Improved error handling during item assignment in pd.eval" in terms of seeing what types of code-blocks to use.